src/main/java/ru/sevsu/rc_manager/sound/handler/SoundHandler.java

```diff
@@ -3,21 +3,24 @@
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Component;
-import ru.sevsu.rc_manager.sound.processor.SoundConverter;
 
-import javax.sound.sampled.AudioInputStream;
+import java.time.Instant;
 
 @Component
 @RequiredArgsConstructor
 @Slf4j
 public class SoundHandler {
     private final SaveSoundHandler saveSoundHandler;
-    private final SoundConverter soundConverter;
+    private final TelegramSendHandler telegramSendHandler;
+    private final RecognizeHandler recognizeHandler;
+
 
-// This handler convert byte array to AudioStream and send it to other handlers
     public void handle(byte[] sound) {
-        log.info("Audio handled");
-        AudioInputStream audioInputStream = soundConverter.byteToStream(sound);
-        saveSoundHandler.handle(audioInputStream);
+        Instant time = Instant.now();
+        log.debug("Audio start handle in " + time);
+        saveSoundHandler.handle(sound);
```

Comment:
that should be an array of autowired handlers that are executed asynchronously

Reply:
I think it will be next step when we make handlers independent, now it will be hard.